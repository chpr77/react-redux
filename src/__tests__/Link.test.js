import React from 'react';
import Link from '../Link'
import renderer from 'react-test-renderer';
  //npm install  react-test-renderer
it('renders correctly', () => {
 const tree = renderer
   .create(<Link page="www.yahoo.com" />)
   .toJSON();
   console.log(tree)
   expect(tree).toMatchSnapshot();
});
