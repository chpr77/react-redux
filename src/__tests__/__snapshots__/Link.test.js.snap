// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <a
    className="normal"
    href="www.yahoo.com"
  >
    www.yahoo.com
  </a>
  Hi there!
</div>
`;
